src/software/vasp.jl

```diff
@@ -359,14 +359,16 @@ readPROCAR(filename::AbstractString) = open(readPROCAR, filename)
 readPROCAR() = open(readPROCAR, "PROCAR")
 
 """
-    getFermi(io::IO) -> Float64
+    get_fermi(io::IO) -> Float64, Float64
```

Comment:
The return type (as is) is a `Tuple{Float64, Float64}`, or equivalently `NTuple{2, Float64}`